src/auth/cephx/CephxProtocol.h

```diff
@@ -18,61 +18,8 @@
 /*
   Ceph X protocol
 
-  First, the principal has to authenticate with the authenticator. A
-  shared-secret mechanism is being used, and the negotitaion goes like this:
+  See doc/dev/cephx.rst
```

Comment:
If you add the anchor to the Cephx doc, you can do this here:

```suggestion
  See :ref:`cephx`
```

Reply:
This code will usually get looked at in an editor, so nothing will render the ref.  I think a path will make it easier to find

Reply:
I totally thought this was an `rst` file! My bad